datafusion/core/tests/parquet/mod.rs

```diff
@@ -925,6 +932,71 @@ fn make_dict_batch() -> RecordBatch {
     .unwrap()
 }
 
+fn make_interval_batch(offset: i32) -> RecordBatch {
+    let schema = Schema::new(vec![
+        Field::new(
+            "year_month",
+            DataType::Interval(IntervalUnit::YearMonth),
+            true,
+        ),
+        Field::new("day_time", DataType::Interval(IntervalUnit::DayTime), true),
+        Field::new(
+            "month_day_nano",
+            DataType::Interval(IntervalUnit::MonthDayNano),
+            true,
+        ),
+    ]);
+    let schema = Arc::new(schema);
+
+    let ym_arr = IntervalYearMonthArray::from(vec![
+        Some(IntervalYearMonthType::make_value(1 + offset, 1 + offset)),
```

Comment:
in general I suggest changing this so the values of the two fields are different (so that it would catch bugs where the fields weren't properly interpreted)

For example, instead of

```rust
        Some(IntervalYearMonthType::make_value(1 + offset, 1 + offset)),
```

Something like (use `10 + offset` in the second field so the values are different)

```rust
        Some(IntervalYearMonthType::make_value(1 + offset, 10 + offset)),
```

The same applies to the rest of the values in this code

Reply:
thank you - fixed